runtime-testsuite/resources/junit-platform.properties

```diff
@@ -1,3 +1,4 @@
 junit.jupiter.execution.parallel.enabled = true
 junit.jupiter.execution.parallel.mode.default = concurrent
-junit.jupiter.execution.parallel.mode.classes.default = concurrent
\ No newline at end of file
+junit.jupiter.execution.parallel.mode.classes.default = concurrent
+junit.jupiter.execution.parallel.config.fixed.parallelism=4
```

Comment:
Setting up a manual value may decrease runtime tests performance.

Also, I don't think `4` is suitable value, modern processors have many thread. I suggest using `8` or at least `6`.

Reply:
This was an oversight. This was not supposed to be pushed, and anyway, it seems to have no effect. I need to find out how to stop the go tests from trying to run every tests at the same time as this takes longer than limiting the threads to the number of cores (or thereabouts). 

@parrt Do you remember how to do this?

Reply:
@jimidle I can't remember what I had for lunch so.... haha